packages/perspective/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.2.0-beta.2",
   "description": "Perspective.js",
   "main": "build/perspective.umd.js",
+  "module": "es/js/perspective.parallel.js",
```

Comment:
AFAIK there's no exclusion for `es/` in .npmignore so it would be published. It's pretty common to separate out the different module formats when publishing a model, e.g. [react](https://unpkg.com/react@16.5.2/) or [rxjs](https://unpkg.com/rxjs) and I think it just provides a but more clarity over which code path users actually end up going down.

Reply:
This module has a [files](https://github.com/jpmorganchase/perspective/blob/f2eb898e13c0ea2cb650c95dca8c0ff361776eac/packages/perspective/package.json#L10) field which exclude 'es/' AFAIK.

Reply:
Logic behind going with `/build` is that this is a complex modules with many shared outputs that are not cleanly one lang or another, and with complex build steps - hence some artifacts going to 'obj' which is also not exported.